Readme.md

```diff
@@ -218,6 +216,30 @@ public class SecondSubCommand : ICommand
 }
 ```
 
+### Cancellation
+
+It is possible to gracefully cancel execution of a command and preform any necessary cleanup. By default an app gets forcefully killed when it receives an interrupt signal (Ctrl+C or Ctrl+Break). Make call to `console.RegisterCancellation()` to override the default behavior and get `CancellationToken` that represents the first interrupt signal. Second interrupt signal terminates an app immediately. 
+
+You can pass `CancellationToken` around and check its state. Make sure you catch `TaskCancelledException`, `OperationCancelledException` with other possible cancellation exceptions.
```

Comment:
I'm not sure I understand 100% what this means: `Make sure you catch `TaskCancelledException`, `OperationCancelledException` with other possible cancellation exceptions.`. Also, I think we don't really need to explain how `CancellationToken` works in general, we can assume our users are intelligent developers. :)

Reply:
Removed. Please check.